src/python/bot/startup/run_heartbeat.py

```diff
@@ -24,6 +24,8 @@
 import subprocess
 import sys
 
+from health_check_responder import run_server as run_health_responser_server
```

Comment:
Can you make this import from the full path (python.bot.startup...) to eliminate any potential import path issues?